homeassistant/components/recorder/db_schema.py

```diff
@@ -588,6 +589,10 @@ def shared_attrs_bytes_from_event(
                 *ALL_DOMAIN_EXCLUDE_ATTRS,
                 *state_info["unrecorded_attributes"],
             }
+            if MATCH_ALL in state_info["unrecorded_attributes"]:
```

Comment:
Maybe put unrecorded_attributes in a local var as well so we don't look it up in the dict twice